technical-resources/operations/open-term-loan-deployment.md

```diff
@@ -0,0 +1,148 @@
+# Phase 1 - Deploy New Contracts
+Deploy and verify the following contracts using a single forge script:
+
+## New PoolDeployer
+1. New PoolDeployer pointing to existing mapleGlobalsProxy
+
+## Globals and PoolManager Upgrade Contracts
+1. Globals v2.0.0 implementation
+2. PoolManager v2.0.0 implementation
+
+## Fixed Term Loan Upgrade Contracts
+1. MapleLoan v5.0.1 implementation
+2. MapleLoanInitializer for v5.0.0 deployments
+3. MapleLoanMigrator for v4.0.0 -> v5.0.0 upgrades
+4. LoanManager v3.0.1 implementation
+
+## Open Term Loan Manager Contracts
+1. LoanManagerFactory
+2. LoanManagerImplementation
+3. LoanManagerInitializer
+
+## Open Term Loan Contracts
+1. MapleLoanFactory
+2. MapleLoanInitializer
+3. MapleLoanImplementation
+4. MapleRefinancer
+
+# Phase 2 - Atomic Configuration of Protocol by Governor
```

Comment:
These steps aren't atomic below. I realize this can be interpreted as "each of these steps dones many things atomically", but it can easily be interpreted as "all these actions are done atomically". I'd drop the word "atomic" here, and make it more clear in each step that the calls are atomic.